src/ariel-os-macros/src/config.rs

```diff
@@ -9,10 +9,11 @@
 ///
 /// - The name of the driver the constant provides configuration for.
 ///
-/// | Driver    | Expected type                  | Cargo feature to enable   |
-/// | --------- | ------------------------------ | ------------------------- |
-/// | `network` | `embassy_net::Config`          | `network-config-override` |
-/// | `usb`     | `embassy_usb::Config`          | `override-usb-config`     |
+/// | Driver    | Expected type                     | Cargo feature to enable   |
+/// | --------- | --------------------------------- | ------------------------- |
+/// | `ble`     | `ariel_os::reexport::ble::Config` | `ble-config-override`     |
```

Comment:
We don't want it to appear in the documentation just yet:
```suggestion
// | `ble`     | `ariel_os::reexport::ble::Config` | `ble-config-override`     |
```